src/Main.java

```diff
@@ -1,5 +1,89 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Exercise 1.");
+        int clientOS = 0;
+        if (clientOS == 0) {
+            System.out.println("Установите версию приложения для iOS по ссылке.");
+        } else {
+            System.out.println("Установите версию приложения для Android по ссылке.");
+        }
+
+
+        System.out.println();
+        System.out.println("Exercise 2.");
+        int clientOSEx2 = 0;
+        int clientDeviceYear = 2015;
+        if (clientOSEx2 == 0 && clientDeviceYear >= 2015) {
+            System.out.println("Установите версию приложения для iOS по ссылке.");
+        } else if (clientOSEx2 == 0 && clientDeviceYear < 2015) {
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке.");
+        } else if (clientOSEx2 == 1 && clientDeviceYear >= 2015) {
+            System.out.println("Установите версию приложения для Android по ссылке.");
+        } else if (clientOSEx2 == 1 && clientDeviceYear < 2015) {
+            System.out.println("Установите облегченную версию приложения для Android" +
+                    " по ссылке.");
+        } else {
+            System.out.println("Параметры введены некорректно.");
+        }
+
+
+        System.out.println();
+        System.out.println("Exercise 3.");
+        int year = 1588400;
+        int year4 = year % 4;
+        int year100 = year % 100;
+        int year400 = year % 400;
+        if (year > 1584 && year4 == 0 && year100 != 0 || year > 1584 && year400 == 0) {
```

Comment:
Можно немного упростить
```suggestion
        if (year > 1584 && (year4 == 0 && year100 != 0 ||  year400 == 0)) {
```